crypto/siphash/siphash_test.cc

```diff
@@ -34,8 +34,8 @@ TEST(SipHash, Basic) {
   for (unsigned i = 0; i < sizeof(input); i++) {
     input[i] = i;
   }
-
-  EXPECT_EQ(UINT64_C(0xa129ca6149be45e5),
+  uint64_t expected = UINT64_C(0xa129ca6149be45e5);
+  EXPECT_EQ(CRYPTO_load_u64_le(&expected),
```

Comment:
Do you know why you need this? I thought creating a variable with a compile time constant value is the same on BE and LE.

Reply:
I was confused. The [SipHash paper](https://cr.yp.to/siphash/siphash-20120918.pdf) explicitly says "k0 and k1 are the little-endian 64-bit words encoding the key k", and I assumed that the resulting hash bytes would also need to be flipped on big-endian; they do not.

The SipHash result **is** the *64-bit-integer-value* (represented in whatever the native byte ordering is) -- the SipHash result is not "bytes with a particular ordering".

I'm updating the logic and the tests for the next commit.